docs/how-to/create-chiselled-docker-image.md

```diff
@@ -0,0 +1,181 @@
+# Create a chiselled Docker image
+
+Since Chisel can produce a minimal root file system, using it in Docker images
+has lots of value. This guide shows how to create a chiselled Docker image.
+
+A _chiselled_ image is one which only contains files installed by the slices and
+does not contain any base images such as the ubuntu image.
+
+
+## Design the image
+
+It's a good idea to think first about what our chiselled image should be able to
+do. Let's consider a Python 3 image, which contains a Python 3 interpreter and
+is able to run Python 3 scripts and commands.
+
+Goals:
+
+- The [image entrypoint] must be `python3`.
+- The image must be chiselled i.e. there must not be anything else.
```

Comment:
see the comment above about the double meaning of the word "chiseled". It's also redundant with the other bullet below about being  "minimal"

Reply:
Removed.